bibtex-actions.el

```diff
@@ -38,6 +38,9 @@
 ;;; Code:
 
 (require 'bibtex-completion)
+(require 'org-element)
```

Comment:
This line can be deleted because `org-element-context` has been declared below

Reply:
I edited your suggestion to use the "suggested" feature, so I could click to commit.

But, as you can see from the CI, I get an error without the require. I think that's why I added it to begin with.

Seems I need to add it back?